modules/ossm-supported-configurations.adoc

```diff
@@ -58,6 +58,7 @@ endif::openshift-rosa[]
 == Supported configurations for Kiali
 
 * The Kiali console is only supported on the two most recent releases of the Chrome, Edge, Firefox, or Safari browsers.
```

Comment:
```suggestion
* The Kiali console is only supported on the two most recent releases of the Google Chrome, Microsoft Edge, Mozilla Firefox, or Apple Safari browsers.
```
- This isn't part of your PR, but technically, the browsers should be referred to with their full names on first mention. SSG link: https://redhat-documentation.github.io/supplementary-style-guide/#mozilla-firefox. Maybe consider fixing this if you have the time while you're working on this file?